docs/website/references/cli/defradb_client_document_purge.md

```
@@ -0,0 +1,51 @@
+Permanently remove documents by DocID from the local node
+
+### Synopsis
+
+Permanently remove a set of documents by DocID from the local node, including all
+datastore values and headstore entries. When --prune-history is set, it also removes
+reachable blockstore blocks that are no longer owned by another document.
+
+Unlike the soft-delete performed by the delete command, this operation is irreversible and
+does not propagate to other nodes in the peer network.
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)